src/JuliaFormatter.jl

```diff
@@ -360,22 +360,16 @@ function format(paths; options...)
             format_file(path; opts...)
         else
             for (root, dirs, files) in walkdir(path)
-                base_opts = if (config = find_config_file(path)) !== nothing
-                    overwrite_options(options, config)
-                else
-                    options
-                end
-
                 for file in files
                     _, ext = splitext(file)
                     ext == ".jl" || continue
                     full_path = joinpath(root, file)
                     ".git" in splitpath(full_path) && continue
-                    dir = relpath(root)
+                    dir = realpath(root)
```

Comment:
the issue was caused by this typo ....
now the config will be detected correctly even if given as relative paths, so we don't need `base_ops` stuff here.